packages/textinput/src/react/index.js

```diff
@@ -122,3 +126,34 @@ TextInput.iconAligns = vars.iconAligns
 export const appearances = vars.appearances
 export const iconAligns = vars.iconAligns
 export default TextInput
+
+function isRef(ref) {
+  if (!isPlainObject(ref)) return false
+
+  return ref.hasOwnProperty('current')
+}
+
+function isRefs(refs) {
+  if (!isPlainObject(refs)) return false
+
+  return Object.keys(refs).every(key => isRef(refs[key]))
+}
+
+function isPlainObject(obj) {
+  return obj && !Array.isArray(obj) && typeof obj === 'object'
+}
+
+function useMultipleRefs(refs) {
+  const hasMultiple = isRefs(refs)
+
+  const prevFieldRef = hasMultiple ? refs['field'] : React.createRef()
```

Comment:
`prev` as in "previous"?  I don't understand what "previous" means in this context.

Reply:
yeah that's confusing. it's the external/forwardeded ref. i'll change